workflow/emc-global/scripts/exglobal_prep_chem.sh

```diff
@@ -163,8 +135,10 @@ for day in $gbbepx_days; do
         export local_name
         for itile in 1 2 3 4 5 6 ; do
             export tiledir=tile$itile
+	    export GTtile=GT$itile
```

Comment:
Such added lines were using tab instead of spaces, you should use spaces to consistent with others.